spec/vcr_cassettes/active_property_successfully_retrieved.yml

```diff
@@ -0,0 +1,57 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://rest.sandbox-immobilienscout24.de/restapi/api/offer/v1.0/user/me/realestate/315661708
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Authorization:
+      - OAuth oauth_consumer_key=myHomedayKey,oauth_nonce=HdfNwSJQ,oauth_signature_method=HMAC-SHA1,oauth_timestamp=1580834887,oauth_token=f7677b9c-c52a-4101-b736-26e8f0e4e898,oauth_version=1.0,oauth_signature=Ki4KnWbVL7sLCFbPGjfWbZqJrvw%3D
```

Comment:
Oh wait, I haven't noticed this. You have to modify this to the Authorization that we have in the other cassetes, to not make public our staging keys. [Like this ](https://github.com/homeday-de/immobilienscout/blob/master/spec/vcr_cassettes/attachment_is_successfuly_created_is24.yml#L23)